### DISCUSSION TITLE
Add a space to collaborately collect interesting resources

### DISCUSSION THREAD

**Discussion by bebatut** [open-life-science/open-life-science.github.io#853] [Ideas and brainstorms]
On the Slack, people are sharing interesting resources, which is great. But it may difficult to keep a track of these resources, specially with the limited Slack history.

It may be interesting to have these resources listed somewhere. They may be useful for the community in general.

What do you think? How could we do that?

One idea (we use this approach in a collaborative blog I manage):
- One Slack channel in which we collect these resources
- A blog post every month(?) which aggregates these resources and spreads them to the world

Any other idea?

**Comment by BatoolMM** [comment]
This will be awesome, it'll be extremely useful if the resources are collected in a separate document. I come through lots of useful links and documents which sometimes I forgot to bookmark in  Open Life Science. Then, it becomes a bit hard to find where it's if I don't remember the exact title especially if the Slack Workplace is very active like **Open Life Science**. If all the resources are added to a separate Slack channel, we might run on the same problem with limited Slack history. **Maybe** they can be added to a google docs/HackMD which is labelled by channel and searchable 😅 . It'd be nice if it indicates that the replication of resources is welcome, so people don't hesitate to add anything. However, this might add an extra layer of burden and complexity. We can also just encourage people to add the resources if they wish (without any pressure) and without anyone maintaining or looking after the file. It might be worth a try. This is my personal opinion, others might not welcome this idea at all.

**Comment by Juxtathinkacode** [comment]
Hello @bebatut I am Gimbiya, Outreachy Intern Applicant and a content writer. Can I be assigned to this task? I would love to create a resourceful blogpost that everyone can contribute to. I can create a Google Doc, update the content and share the link on Slack.

**Comment by SaraVilla** [comment]
what about :
- posting the resources share in the Slack canvas: those are static I believe
- make a GH table and collate resources there?
